content/en/docs/setup/install/operator/index.md

```diff
@@ -75,25 +73,214 @@ accessible to `istioctl` by using this command:
+
+Features can be enabled or disabled, which enables or disables all of the components that are a part of the feature.
+Namespaces that components are installed into can be set by component, feature, or globally (see example in next
+section).
+
+### Configure the feature or component settings
+
+After you identify the name of the feature or component from the previous table, you can use the API to set the values
+using the `--set` flag, or create an overlay file and pass it with the `--filename` flag. The `--set` flag
+works well for customizing a few parameters, while overlay files are better for more extensive customization, or
+tracking configuration changes.
+
+The simplest customization is to turn a feature or component on or off from the configuration profile default.
+
+For example, to disable the telemetry feature in a default configuration installation, use this command:
+
+{{< text bash >}}
+$ istioctl experimental manifest apply --set telemetry.enabled=false
+{{< /text >}}
+
+Alternatively, you can disable the telemetry feature using a configuration overlay file:
+
+1. Create this file with the name telemetry_off.yaml and these contents:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  telemetry:
+    enabled: false
+{{< /text >}}
+
+1. Provide the telemetry_off.yaml overlay file to `manifest apply` command:
 
 {{< text bash >}}
-$ istioctl experimental profile list -s installPackagePath=https://github.com/istio/istio/releases/tags/1.3.3
+$ istioctl experimental manifest apply -f telemetry_off.yaml
+{{< /text >}}
+
+You can also use this approach to set the component-level configuration, such as enabling the node agent:
+
+{{< text bash >}}
+$ istioctl experimental manifest apply --set security.components.nodeAgent.enabled=true
+{{< /text >}}
+
+Another customization is to select different namespaces for features and components. The following is an example
+of installation namespace customization:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  defaultNamespace: istio-system
+  security:
+    namespace: istio-security
+    components:
+      citadel:
+        namespace: istio-citadel
+{{< /text >}}
+
+Applying this file will cause the default profile to be applied, with components being installed into the following
+namespaces:
+
+- Citadel component installed into istio-citadel namespace
+- All other components in the security feature installed into istio-security namespace
+- Remaining Istio components installed into istio-system namespace
+
+## Customize K8s settings using the IstioControlPlane API
+
+The IstioControlPlane API allows each component's K8s settings to be customized in a consistent way.
+
+### Identify the feature or component settings
+
+Each component has a [KubernetesResourceSpec](https://github.com/istio/operator/blob/9f80ecaea0f17dfd8a33d86d72f72da8861e7417/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto#L411),
```

Comment:
is there any way to link this not using github?  The istiocontrolplane proto is in the docs right?  Shouldn't KubernetesResourceSpec have its own link target in that document?

Reply:
Done.